app/css/openmrs-addonmanager.css

```diff
@@ -825,3 +825,7 @@ body.loading .waiting-modal {
     font-size: 1.0em;
     padding: 5px;
 }
+
+.main-row {
+    width: 50vh;
+}
```

Comment:
since you are using width isn't the unit supposed to be `vw` ??
cause `vh` is the unit for height.

Reply:
👍

Reply:
Thanks, i have updated the PR